devops/templates/post-upload-validation.yml

```diff
@@ -0,0 +1,63 @@
+# Stage template to validate uploaded packag from PyPI
+
+# Downloads fairlearn from the specified PyPI (Test or Prod) at the version
+# specified in the Version Artifact. Then runs tests against that install
+
+parameters:
```

Comment:
I wonder whether it's simpler not to have this as a template. Briefly touched on this with @eedeleon today and we've created such a high barrier to entry with all the templating that I'd actually take a step back and perhaps remove all templates that aren't used multiple times. We have 30 lines of input parameter spec here for 30 lines of actual logic.

Reply:
The idea was to make the top level file as clean as possible, so the individual stages could be seen clearly. I agree that mustache templating is not the nicest, though.